assets/aui.selector.publish.js

```diff
@@ -14,15 +14,61 @@
 			fields.each(buildInterface);
 		};
 
+		var updateFilters = function(id,newfilters) {
+			fields.each(function(){
+				var field = $(this); 
+				if (field.attr('id') == "field-" + id){
+					field.data("filters",newfilters);
+					var storage = field.find('.selectized'),
+						selectize = storage[0].selectize,
+						fieldId = field.data('parent-section-field-id'),
+						limit = parseInt(field.data('limit')),
```

Comment:
Never call `parseInt` without a proper suffix. I suppose you use base 10 here so `parseInt(field.data('limit'), 10)`